e2e/pvc.go

```diff
@@ -244,3 +245,44 @@ func checkPVSelectorValuesForPVC(f *framework.Framework, pvc *v1.PersistentVolum
 	}
 	return nil
 }
+
+func getMetricsForPVC(f *framework.Framework, pvc *v1.PersistentVolumeClaim, t int) error {
```

Comment:
as we randomly picking a node IP here will this work, if we don't have any pods running on this one? If not add a FIXME here?